File: java/src/main/java/com/languageweaver/sdk/samples/dictionaries/EdgeDictionaryService.java
package com.languageweaver.sdk.samples.dictionaries;

import com.languageweaver.sdk.common.edge.EdgeLanguageWeaverClient;
import com.languageweaver.sdk.dictionaries.edge.result.EdgeDictionariesResult;

public class EdgeDictionaryService {
    public static final int PAGE_NUMBER = 1;
    public static final int PAGE_SIZE = 100;

    public static void main(String[] args) throws Exception {
        try (EdgeLanguageWeaverClient lwClient = new EdgeLanguageWeaverClient().build()) {
            final EdgeDictionariesResult dictionariesResult = lwClient.getEdgeDictionaries(PAGE_NUMBER, PAGE_SIZE);

            if (dictionariesResult.getDictionaries() != null) {
                dictionariesResult.getDictionaries()
                        .forEach(dictionary -> System.out.println(dictionary.getDictionaryId()));
            }
        }
    }
}

File: dotnet/Samples/Dictionaries/EdgeDictionaryService.cs
using LanguageWeaver.Sdk.Common.Edge;
using LanguageWeaver.Sdk.Dictionaries.Edge.Result;

namespace LanguageWeaver.Sdk.Samples.Dictionaries;

public static class EdgeDictionaryService
{
    public static readonly int PAGE_NUMBER = 1;
    public static readonly int PAGE_SIZE = 100;

    public static void Main()
    {
        using var lwClient = new EdgeLanguageWeaverClient().Build();
        EdgeDictionariesResult dictionariesResult = lwClient.GetEdgeDictionaries(PAGE_NUMBER, PAGE_SIZE);

        dictionariesResult.Dictionaries
            .ForEach(dictionary => Console.WriteLine(dictionary.DictionaryId));
    }
}